server/src/websocket_uploader/engine_uploader.c

```diff
@@ -170,8 +170,8 @@ void process_verification_result_msg(struct mg_ws_message *ws_msg, ws_verif_data
 	char *verif_success = strstr(ws_msg->data.ptr, msg_type_result_status_true_sig);
 	
 	if (verif_success) {
-		syslog_server(LOG_INFO, "Websocket upload engine: Verification Server finished,"
-		              " verification succeeded");
+		syslog_server(LOG_INFO, 
+		              "Websocket upload engine: Verification Server done, verification succeeded");
```

Comment:
> "Websocket upload engine: uploaded engine satisfies all its verification properties"

I'll think about all the logs in that function again, think they should be somewhat consistent. The aspect of the verification server being "done" could also be extracted to a separate log to avoid repitition.